### DISCUSSION TITLE
Modify scope of bug reporting

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5583] [Feature Requests]
Security-conscious institutions will want to control the scope of data shared in an error report

**Comment by maxpatiiuk** [comment]
List of things that are currently included in the crash report along with an error message:

- Current user
- System information (sp7 version, sp6 version, schema version, guids of scoping levels)
- Schema configuration
- Current URL
- Log of all console messages and errors since the page was opened
- List of all resources that the webpage loaded and how long it took to load them
- Permissions of the current user
- Remove prefs
- User preferences
- UI Language
- User agent string (it identifies current browser version and OS)
- User's preferred languages (set in browser preferences)

Need to discuss with the team if admins should be able to exclude any of this from the crash reports.

Keep in mind though, the crash report is just a regular JSON file that can be edited in any text editor. They can redact the confidential information out of it before sending it to us.
